pkg/cvo/sync_worker.go

```diff
@@ -1050,6 +1076,9 @@ func (r *consistentReporter) Update() {
 	r.reporter.Report(copied)
 }
 
+// Errors updates the status based on the current state of the graph runner.
+//
+// SyncWorker r.reporter.w must not be locked before Update is called.
```

Comment:
"before Update is called" looks like a copy/paste issue?  I'd expect "before Errors is called" here, and similarly for `Completed` below.